.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build:
+    environment:
+      CC_TEST_REPORTER_ID: bdda8062e987a067dcdfe953dbc12024acaeb7e4c621304b69ba725fb5470a3e
+
+    working_directory: ~/slovensko-digital/www.ecosystem
+
+    docker:
+      - image: circleci/ruby:2.5.0-node-browsers
+        environment:
+          RAILS_ENV: test
+
+      - image: circleci/postgres:9.6.8-alpine-ram
+        environment:
+          POSTGRES_USER: ubuntu
+          POSTGRES_PASSWORD: ''
+
+    steps:
+      - checkout
+      - run: curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./test-reporter
```

Comment:
Preco si toto vlastne presunul z gemu do CI?

Reply:
soon to be deprecated https://github.com/codeclimate/ruby-test-reporter
takto je to po novom https://github.com/codeclimate/test-reporter

celkovo konfiguracia pre Circle CI 2.0 je o dost ukecanejsia